open_lm/main.py

```diff
@@ -355,6 +362,10 @@ def main(args):
         resume_from = None
         checkpoint_path = args.checkpoint_path
 
+        # If using remote_sync, need to check the remote instead of the local checkpoints folder.
+        if args.remote_sync is not None:
+            checkpoint_path = os.path.join(args.remote_sync, args.name, "checkpoints")
```

Comment:
One question here @sagadre - we assign checkpoint_path, but we don't update args.checkpoint_path. I copied this from code from before #153. Should we just go ahead and update args.checkpoint_path too? Not sure if we use it anywhere else.

Reply:
looked over the code and this should work and not break other things. my understanding is checkpoint_path is used just to load the checkpoint, e.g., latest checkpoint from s3; however, in general we probably should simplify the checkpoint loading in a separate PR. right now it is a bit confusing with checkpoint_path, args.checkpoint_path, join(.., checkpoints), remote-sync, etc.

Reply:
maybe just having one factorized helper function to do all this loading business could be nice and also more easily unit-testable